manifests/0000_12_etcd-operator_01_config.crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: etcds.operator.openshift.io
+spec:
+  scope: Cluster
+  group: operator.openshift.io
```

Comment:
I just wonder if we should make this EtcdCluster isn't Cluster `clusters.cluster.k8s.io`?

Reply:
> I just wonder if we should make this EtcdCluster isn't Cluster `clusters.cluster.k8s.io`?

`oc get etcd.operator.openshift.io`